themes/bootstrap3/templates/admin/feedback/home.phtml

```diff
@@ -0,0 +1,153 @@
+            </td>
+            <td><?=$this->escapeHtml($feedbackItem->site_url)?></td>
+            <td><?=$this->escapeHtml($feedbackItem->message)?></td>
+            <td>
+              <?php if (!empty($data)): ?>
+                <a href="#" class="btn btn-default btn-info" data-toggle="modal" data-target="#feedback-<?=$feedbackItem->id?>">
+                  <?=$this->transEsc('Show')?>
+                </a>
+                <div class="modal" id="feedback-<?=$feedbackItem->id?>" tabindex="-1" role="dialog" aria-hidden="true">
+                  <div class="modal-dialog">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
+                          <?=$this->icon('lightbox-close', ['aria-label' => $this->translate('Close')])?>
+                        </button>
+                        <h4 class="modal-title"><?=$this->transEsc('Additional data')?></h4>
+                      </div>
+                      <div class="modal-body">
+                        <?php foreach ($data as $key => $value): ?>
+                          <strong><?=$this->escapeHtml($key)?></strong>: <?=$this->escapeHtml($value)?><br>
+                        <?php endforeach; ?>
+                        <strong><?=$this->transEsc('Created')?></strong>:
+                        <?=$this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime('Y-m-d H:i:s', $feedbackItem->created))?>
+                        <?=$feedbackItem->user_name ? (' ' . $this->transEsc('by') . ' ' . $this->escapeHtml($feedbackItem->user_name) . ' (' . $feedbackItem->user_id . ')'): ''?>
+                        <br>
+                        <strong><?=$this->transEsc('Updated')?></strong>:
+                        <?=$this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime('Y-m-d H:i:s', $feedbackItem->updated))?>
+                        <?=$feedbackItem->manager_name ? (' ' . $this->transEsc('by') . ' ' . $this->escapeHtml($feedbackItem->manager_name) . ' (' . $feedbackItem->updated_by . ')') : ''?>
+                      </div>
+                    </div>
+                  </div>
+                <?php endif; ?>
+            </td>
+            <td>
+              <select class="form-control status_update" name="status_update">
+                <?php foreach ($this->statuses as $status): ?>
+                  <option value="<?=$this->escapeHtmlAttr($status)?>"<?=$status == $feedbackItem->status ? ' selected="selected"' : ''?>>
+                    <?=$this->transEsc('feedback_status_' . $status)?>
+                  </option>
+                <?php endforeach; ?>
+              </select>
+            </td>
+          </tr>
+        <?php endforeach; ?>
+      </table>
+      <input type="submit" name="deleteSelected" value="<?=$this->transEscAttr('delete_selected')?>" class="btn btn-default">
```

Comment:
I realize that this is another detail inherited from the existing Tag functionality, but I wonder if duplicating these buttons at the top would make the form more user-friendly.